src/lib/CMakeLists.txt

```diff
@@ -610,6 +611,10 @@ set(
     ${CMAKE_DL_LIBS}
 )
 
+if (NOT APPLE)
+    set(LIBRARIES ${LIBRARIES} atomic)
```

Comment:
Could we add a comment here?

Reply:
Not sure what to write here. Linux wants `-latomic`, Apple doesn't. But that's what's written there.

Reply:
I didn't know that libatomic was a thing.

Reply:
Neither did I, and I still don't understand why it still became necessary. The GCC documentation says "Linking to libatomic is required for some uses of ISO C++11 <atomic>. ". Very helpful, thank you.